dropboxfs.py

```diff
@@ -133,6 +133,7 @@ def __init__(self, client, name):
         self.closed = False
         self.pos = 0
         self.seek_pos = 0
+        super(ChunkedReader, self).__init__(self.r, name)
```

Comment:
It requires self.r to be set, which means it would at least have to be in the middle of the function anyway. And this also follows the pattern of the other similar classes that inherit from the same place